src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap

```diff
@@ -0,0 +1,35 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InstructorSearchPageComponent should snap with a student table 1`] = `
+<tm-instructor-search-page
+  fbSessionDataTables={[Function Array]}
+  httpRequestService={[Function HttpRequestService]}
+  route={[Function ActivatedRoute]}
+  searchKey=""
+  statusMessageService={[Function StatusMessageService]}
+  studentTables={[Function Array]}
+  user="undefined"
```

Comment:
This too

Reply:
this value also does not seem to have a use

Reply:
Ah yup don't need